src/pages/EditItemPage.js

```diff
@@ -71,23 +71,8 @@ const EditItemPage = () => {
     }
   };
 
-  const handleFileUpload = async (e) => {
-    try {
-      if (e.target.files.lenght !== 0) {
-        setTempImageUrl(URL.createObjectURL(e.target.files[0]));
-
-        let resizedImg = await convert({
-          file: e.target.files[0],
-          width: 300,
-          height: 225,
-          type: 'jpeg',
-        });
-
-        setObjImageToUpload(resizedImg);
-      }
-    } catch (error) {
-      console.log('Error while uploading the file: ', error);
-    }
+  const handleImageUpload = async (e) => {
```

Comment:
Why do you need to create an async function to execute just one line? I think it would be better if you don´t break in function in this case